core/src/main/java/org/ehcache/core/spi/cache/AbstractValueHolder.java

```diff
@@ -109,7 +124,16 @@ public long lastAccessTime(TimeUnit unit) {
   }
 
   public void setLastAccessTime(long lastAccessTime, TimeUnit unit) {
-    this.lastAccessTime = unit.convert(lastAccessTime, nativeTimeUnit());
+    lastAccessTime = unit.convert(lastAccessTime, nativeTimeUnit());
```

Comment:
Removing the `this.` does not magically create a local var. You are still setting it blindly.